src/cve/data_models/cve_intel.py

```diff
@@ -174,6 +217,24 @@ class CveIntel(BaseModel):
     ubuntu: CveIntelUbuntu | None = None
     epss: CveIntelEpss | None = None
 
+    @computed_field()
+    @property
+    def sufficient_intel(self) -> bool:
```

Comment:
Minor nit on naming:
```suggestion
    def has_sufficient_intel(self) -> bool:
```